src/components/AssetsFilters/index.jsx

```diff
@@ -20,7 +18,7 @@ export const AssetsFilters = ({ assetsFilters, updateFilter, updatePage }) => (
             name={type.key}
             label={type.displayName}
             checked={assetsFilters.assetTypes[type.key]}
-            onChange={(checked) => { updatePage(0); updateFilter(type.key, checked); }}
+            onChange={(checked) => { updateFilter(type.key, checked, courseDetails); }}
```

Comment:
The action creator now handles updating the page, so the component no longer needs to do that.